docs/eventlisteners.md

```diff
@@ -257,7 +263,39 @@ spec:
 
 With the help of `kubernetesResource` user can specify [PodTemplateSpec](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L3704).
 
-Right now the allowed values as part of `podSpec` are
+#### CustomResource
+`CustomResource` have one field which supports dynamic object
+* runtime.RawExtension
+
+Used Knative Service as an example to demonstrate usage of `CustomResource`
+
+**Note:** `Knative Should be installed on the cluster` [ref](https://github.com/tektoncd/community/blob/main/teps/0008-support-knative-service-for-triggers-eventlistener-pod.md#note)
+```yaml
+spec:
+  resources:
+    customResource:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      metadata:
+        name: knativeservice
```

Comment:
do we need to provide a name? Shouldn't we follow the same naming convention as we do for the k8s deployment i.e the el name with the `el-` prefix?

Reply:
If name provided  it uses that or else `EL` uses name with the `el-` prefix 

But to keep consistency i will just remove name field from example yaml

Reply:
yeah if we are using a different name, we also need to change the value in status in status.configuration.generatedName